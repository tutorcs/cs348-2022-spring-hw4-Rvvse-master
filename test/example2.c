https://tutorcs.com
WeChat: cstutorcs
QQ: 749389476
Email: tutorcs@163.com
#include "homework.h"

int main() {
  int x = source();
  int y;
  if (x > 0) {
    y = sanitizer(x);
    sink(y); // safe
  } else {
    y = x;
  }
  sink(y); // bug
  return 0;
}
