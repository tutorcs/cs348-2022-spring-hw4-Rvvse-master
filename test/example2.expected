Potential Error @ example2.c:4 (example2.c:12)
